src/main/java/com/johnstarich/moviematcher/app/MovieMatcherApplication.java

```diff
@@ -72,6 +82,16 @@ public void loginService() {
 			return "passwords don't match";
 		});
 
+		jget("/login", (request, response) -> {
+			Optional<String> session_id = Optional.ofNullable(request.headers("Authorization"));
+
+			if(! session_id.isPresent()) throw new HttpException(HttpStatus.UNAUTHORIZED, "No session");
```

Comment:
`"Invalid session"` is better since it doesn't give away that the session has expired (potential security hole)